LDAR_Sim/src/methods/company.py

```diff
@@ -253,15 +255,14 @@ def flag_site(self, site):
                 self.name)][self.state['t'].current_timestep] += 1
 
             # Check to see if the site has any leaks that are active and tagged
-            site_leaks = [lk for lk in self.state['leaks']
-                          if lk['status'] == 'active' and lk['tagged']
-                          and lk['facility_ID'] == site_obj['facility_ID']]
-            if len(site_leaks) > 0:
+            site_leaks = sum([lk for lk in site_obj['active_leaks'] if lk['status'] == 'tagged'])
```

Comment:
Again, not sure if this was the point of "status" - can't you check wehther each leak is tagged using the 'tagged' variable?

Reply:
Actually, yeah this is a good idea, status and tagged.